charts/rstudio-workbench/files/job.tpl

```diff
@@ -77,6 +83,7 @@ spec:
       {{- if .Job.host }}
       nodeName: {{ toYaml .Job.host }}
       {{- end }}
+      enableServiceLinks: {{ if hasKey $templateData.pod "enableServiceLinks" }}{{ $templateData.pod.enableServiceLinks }}{{ else }}false{{ end }}
```

Comment:
@jspiewak What do you think about this?

"Proper" backwards compatibility would mean leaving this unset if it is not specified, but this is a better security posture so you do not expose the endpoints of your other services on Kubernetes to arbitrary Connect content.

Reply:
I can't speak for workbench but for connect I think it's fair to say that these content pods are an implementation detail and shouldn't be considered a part of our public API. I'm fine with changing the default in Connect

Reply:
What is the behavior when unset?
What would a customer experience with this unexpectedly defaulting to `false`?

Reply:
So, you could roll this out in two steps:
1. Enable control over the setting, but maintain existing behavior
2. Move to defaulting to `false`

Two steps is more work, but gives consumers additional control.
Not clear to me if it is worth it. What is the likelihood that anyone is actually relying upon it being true?

Yes, very interesting discussion. Might be worth linking to in the NEWS item you have.

Reply:
I think it's extremely unliklely that anyone is relying upon it being true. And I like the idea of just flipping the switch so (1) maybe we get some data if they are, (2) we ensure everyone has a "more secure" / more stable environment.

I'll definitely add to NEWS!